fix: gst profile env clean up and yolov5s_efficientnet no hardcoded var

vli11 opened the pull request:
yolov5-cpu.env and yolov5-gpu.env clean up and yolov5s_efficientnet use env variable values

Closes: #410

## PR Checklist
<!-- Please check if your PR fulfills the following requirements: -->

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
<!-- Please provide a short description of the updates that are in the PR -->

## Issue this PR will close

close: #410 

## Anything the reviewer should know when reviewing this PR?
tested with integrated gpu
tested with cpu (core)
tested with serpent canyon dgpu
tested yolov5s.sh, yolov5s_full.sh, yolov5s_effnetb0.sh with existing env files with platform core, dgpu, dgpu.1

## Test Instructions if applicable
1. edit configs/opencv-ovms/cmd_client/res/gst/configuration.yaml, update PipelineInputArgs: "--pipeline_script_choice yolov5s_effnetb0.sh"
2. start the pipeline with cmd: PIPELINE_PROFILE="gst" RENDER_MODE=0 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )

jim-wang-intel commented on configs/opencv-ovms/gst/framework-pipelines/yolov5_pipeline/yolov5s_effnetb0.sh:
```diff
@@ -11,11 +11,11 @@ if [ "$INPUTSRC_TYPE" == "REALSENSE" ]; then
 echo "Run run yolov5s with efficientnet classification pipeline"
-gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect model-instance-id=odmodel name=detection model=models/yolov5s/1/FP32-INT8/yolov5s.xml model-proc=models/yolov5s/1/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS $DETECTION_OPTIONS ! gvatrack name=tracking tracking-type=zero-term-imageless ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=0 ! gvaclassify model-instance-id=clasifier labe ...
\ No newline at end of file
+gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect model-instance-id=odmodel name=detection model=/home/pipeline-server/models/yolov5s/FP16-INT8/1/yolov5s.xml model-proc=/home/pipeline-server/models/yolov5s/FP16-INT8/1/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS $DETECTION_OPTIONS ! gvatrack name=tracking tracking-type=zero-term-imageless ! queue max-size-bytes=0 max-size-buffers=0 max-size ...
```
could you also make it consistent with other two having bash_cmd and echo it out?

jim-wang-intel reviewed (commented):
Do we have any changes to env files? if so, please check them in.

vli11 commented:
> Do we have any changes to env files? if so, please check them in.

yes, please see 'tested' for reviewer to know.

jim-wang-intel commented:
> > Do we have any changes to env files? if so, please check them in.
> 
> yes, please see 'tested' for reviewer to know.

I don't understand what you meant.  I was saying any changes to yolov5-cpu.env and yolov5-gpu.env files. but i am only seeing sh script file changes [on the file changes](https://github.com/intel-retail/automated-self-checkout/pull/415/files).

antoniomtz reviewed (approved):
LGTM

jim-wang-intel reviewed (changes requested):
issue #410 saying yolov5-cpu.env and yolov5-gpu.env clean up but i dont' see any changes. this is confusing

vli11 commented:
> issue #410 saying yolov5-cpu.env and yolov5-gpu.env clean up but i dont' see any changes. this is confusing

I fixed #410 according to the description, sorry for the confusion, I've updated the issue title on #410.
